### PR TITLE
mco-fcos: Publish release image to origin namespace

### PR DESCRIPTION
**Pull request by LorbusChris**
Testing...

### CODE REVIEW

File: ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-fcos-presubmits.yaml [hunk 1]
```diff
@@ -142,6 +141,7 @@ presubmits:
         - --resolver-address=http://ci-operator-configresolver
         - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
         - --target=[images]
+        - --target=[release:latest]
```
Comment (LorbusChris) [inline ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-fcos-presubmits.yaml]: Where do I add this config so it shows up here when re-generating?

Comment (LorbusChris) [reply ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-fcos-presubmits.yaml]: for comparison, on the master branch, we have it here: https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml#L432

Comment (smarterclayton) [reply ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-fcos-presubmits.yaml]: you have to change openshift/ci-tools ci-operator-prowgen to correctly do this.  It currently assumes "ocp" is the only target promotion tag image stream name.

Comment (LorbusChris) [reply ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-fcos-presubmits.yaml]: hold on, so this *is* actually relevant here?

**Pull request closed without merging**